api/commands/run.ts

```diff
@@ -42,7 +42,10 @@ export class Runner {
   private readonly runResult: dataform.IRunResult;
   private readonly changeListeners: Array<(graph: dataform.IRunResult) => void> = [];
   private readonly eEmitter: EventEmitter;
-
+  private readonly flags: {
```

Comment:
You want to keep the flags at the top level, and then use them as defaults - is probably a cleaner way to do this! (And keep your new flag added)